source/languages/en/riak/ops/advanced/backends/multi.md

```diff
@@ -99,7 +99,7 @@ Here is an example of the general form for configuring multiple
 backends:
 
 ```riakconf
-multi_backend.$name.$setting_name = setting
+multi_backend.$name.backend_name.$setting_name = setting
```

Comment:
I would say that we tend to think of settings in riak.conf as their whole dotted name representation, `bitcask.io_mode` for example.  So in this case, I think this edit might be overreaching.